src/Form/inputTypeConfig.js

```diff
@@ -42,10 +42,26 @@ const inputTypeConfig = {
         getIconFilled: () => '',
         shouldDecorate: (isLogin, device) => device.hasLocalCreditCards,
         dataType: 'CreditCards',
-        displayTitlePropName: 'title',
+        displayTitlePropName: (input, data) => data.title,
         displaySubtitlePropName: 'displayNumber',
         autofillMethod: 'getAutofillCreditCard'
     },
+    identities: {
+        type: 'identities',
+        getIconBase: () => '',
+        getIconFilled: () => '',
+        shouldDecorate: (isLogin, device) => device.hasLocalIdentities,
+        dataType: 'Identities',
+        displayTitlePropName: (input, data) => {
```

Comment:
I've introduced these functions specifically for the identities, because we show different things in the UI based on the input field.